services/slam/server_test.go

```diff
@@ -170,11 +170,11 @@ func TestWorkingServer(t *testing.T) {
 		poseSucc := spatial.NewPose(r3.Vector{X: 1, Y: 2, Z: 3}, &spatial.OrientationVector{Theta: math.Pi / 2, OX: 0, OY: 0, OZ: -1})
 		componentRefSucc := "cam"
 
-		injectSvc.GetPositionFunc = func(ctx context.Context) (spatial.Pose, string, error) {
+		injectSvc.PositionFunc = func(ctx context.Context) (spatial.Pose, string, error) {
 			return poseSucc, componentRefSucc, nil
 		}
 
-		injectSvc.GetPointCloudMapFunc = func(ctx context.Context) (func() ([]byte, error), error) {
+		injectSvc.PointCloudMapFunc = func(ctx context.Context) (func() ([]byte, error), error) {
 			reader := bytes.NewReader(pcd)
 			serverBuffer := make([]byte, chunkSizeServer)
 			f := func() ([]byte, error) {
```

Comment:
Check comments on line 190 and 203

Reply:
Yes

Reply:
I assumed that the comments are referring to the server functions, which are still `GetPosition` and `GetPointCloudMap`, and thus don't need to be changed.

Reply:
Good point! Ignore my suggestion